hoge.rb

```diff
@@ -0,0 +1,2 @@
+a =  aa
```

Comment:
__[development][rubocop]__
* Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->